### ISSUE TITLE
build failure on Scala 2.12.0-RC1 because Range.Inclusive is final now

### ISSUE THREAD

**Issue by SethTisue** [akka/akka#21548]
```
[akka-http] [error] /Users/tisue/community-builds.212/target-0.9.5/project-builds/akka-http-2993527e43e49e6d7b8326a319451cfd3000db8b/akka-stream/src/main/scala/akka/stream/javadsl/Source.scala:159: illegal inheritance from final class Inclusive
[akka-http] [error]         new Inclusive(start, end, step) {
[akka-http] [error]             ^
```

as discussed at https://github.com/scala/community-builds/issues/146

**Comment by SethTisue** [comment]
looks like this should be closed now that #21566 was merged?

**Comment by patriknw** [comment]
Thx
